Bash (AWS).sublime-syntax

```diff
@@ -0,0 +1,25 @@
+%YAML 1.2
+---
+# See http://www.sublimetext.com/docs/3/syntax.html
+scope: source.shell.bash.aws
+version: 2
+hidden: true
+name: YAML Sequence delimited Bash
```

Comment:
That's the first hidden syntax explicitly specifiying `name`. As it is not listed anywhere, is it needed?

Reply:
It's not needed, I was mainly using it as a reminder to myself that it's a slightly unusual syntax embedding. I should change the base scope or have it as a comment instead 🙂 I will do so before merging

Reply:
It doesn't hurt, if present. Maybe just adjust position.

Reply:
Imo using the `name` main key in any way is mostly confusing because it will raise the question of which name a syntax-specific settings file should have because it's always the syntax's file name and not the name shown in the status bar (which is sourced from this `name` key).

Now, this is a hidden embedded syntax, but I would still suggest to never use the key to not set a bad example.

Reply:
removed the name key, made it a comment :+1: